package.json

```diff
@@ -896,10 +900,15 @@
                     "group": "inline@1"
                 },
                 {
-                    "command": "aws.s3.downloadFileAs",
+                    "command": "aws.s3.openFile",
                     "when": "view == aws.explorer && viewItem == awsS3FileNode",
                     "group": "inline@1"
                 },
+                {
+                    "command": "aws.s3.downloadFileAs",
+                    "when": "view == aws.explorer && viewItem == awsS3FileNode",
+                    "group": "inline@2"
```

Comment:
why not keep it in the same group? `inline@1`

too many groups makes groups meaningless.

Reply:
It was done like that so that that `downloadFileAs` appear second, to keep consistency with uploadFile cloud icon. If it is changed to inline@1, it will appear first, then the icon of openFile will come in